package.json

```
@@ -4,6 +4,7 @@
 		"dev": "cross-env NODE_OPTIONS='--inspect' next dev",
 		"build": "next build",
 		"start": "next start",
+		"type-check": "tsc --noEmit",
```

Just to keep it consistent with other npm scripts name
```suggestion
		"type:check": "tsc --noEmit",
```